### PR TITLE
EP024 blueprint - Managed resources consistency checks

### PR DESCRIPTION
**Pull request by viniarck**
@jab1982 @italovalcy @ajoaoff, I've captured the ideas, potential solutions and new requirements that the team has brainstormed and discussed so far. I'll set this PR as draft to continue the collaboration and collecting feedback. 

The rendered preview [can be seen on this file on my branch](https://github.com/viniarck/kytos/blob/feature/ep24/docs/blueprints/EP024.rst)

Thanks.

### CODE REVIEW

**Comment by viniarck** [comment]
I'm removing the draft status, I've pushed some recent commits based, it's ready for review.

File: docs/blueprints/EP024.rst
Comment (italovalcy) [inline docs/blueprints/EP024.rst:20]:
```suggestion
This document will describe some existing shortcomings of the current consistency checks on `flow_manager <https://github.com/kytos-ng/flow_manager>`_, and propose some improvements and also general development and usage guidelines. This document extends the `previous Kytos consistency documentation  <https://docs.kytos.io/developer/consistency/>`.
```

**Review by italovalcy — CHANGES_REQUESTED** [review]

**Comment by italovalcy** [comment]
The documentation is very well written and provides a great level of details/examples. Thanks for documenting this blueprint, @viniarck.

File: docs/blueprints/EP024.rst
Comment (viniarck) [reply docs/blueprints/EP024.rst]: Excellent point. It'll be handy for future readers. Just committed the suggested patch.

**Comment by viniarck** [comment]
> The documentation is very well written and provides a great level of details/examples. Thanks for documenting this blueprint, @viniarck.

The brainstorm session we had was really great, it helped a ton and expedited the decision making, thanks for contributing and reviewing @italovalcy. Once the PR is approved, after that, I'm planning to squash the commits into a single one since it's just one doc file, if that's OK.

**Review by italovalcy — APPROVED** [review]

**Pull request merged**
